assets/openflamingo.yaml

```diff
@@ -0,0 +1,25 @@
+---
+
+- type: model
+  name: OpenFlamingo
+  organization: LAION
+  description: An open-source reproduction of DeepMind's Flamingo model. At its
+    core, OpenFlamingo is a framework that enables training and evaluation of large
+    multimodal models (LMMs).
+  created_date: 2023-03-28
+  url: https://laion.ai/blog/open-flamingo/
+  model_card: https://github.com/mlfoundations/open_flamingo/blob/main/MODEL_CARD.md
+  modality: image and text input with textual output
+  analysis: Evaluated on COCO captioning and VQAv2 vision-language tasks.
+  size: 9B parameters
+  dependencies: [Flamingo, LLaMA]
```

Comment:
I don't think this is right. In particular, the reason OpenFlamingo was built is because Flamingo is not available, so this doesn't quite make sense. 

As I mention above, make sure the dependencies are concrete assets (e.g. models) that are used, not abstract techniques/methods.